basic_module/account/src/traits/core.rs

```diff
@@ -0,0 +1,36 @@
+pub mod handles {
+    pub use crate::account::AccountError;
+    pub use crate::Transaction;
+    pub use coordinator::context::SubStorageAccess;
+    pub use key::{Address, Error as KeyError, Password, Private, Public, Signature};
+
+    pub trait CheckTxHandler {
+        fn verify_transaction(tx: &Transaction) -> bool;
+    }
+
+    pub trait TransactionExecutor {
+        fn execute_transactions(transactions: &[Transaction]);
+    }
+
+    pub trait AccountManager {
+        fn add_balance(address: &Address, val: u64);
```

Comment:
Foundry module system requires interfaces to have (&self) parameter because it utilizes `trait Object`.

Reply:
I've fixed it on previous modification, I didn't update it. Thanks. I just updated it.